gui/src/customSwitch.cpp

```diff
@@ -130,4 +130,28 @@ void CustomSwitch::showEvent(QShowEvent *event)
 
 }
 
+const QLabel &CustomSwitch::getOn() const {
+        return on;
+}
+
+void CustomSwitch::setOn(const QLabel &on_) {
+        this->on.setText(on_.text());
```

Comment:
This method could be renamed to getOnText and setOnText
You could then use setOnText(QString) to set the text if necessary.

Same goes for setOff.